src/utils/getReactNativeVersion.js

```diff
@@ -7,18 +7,21 @@
 
 const resolveCwd = require('resolve-cwd');
 const fs = require('fs');
+const path = require('path');
 const logger = require('../logger');
 
 /**
 * Search top-most node_modules for React Native
 * and return it's version
 */
 
-module.exports = (): string => {
+module.exports = (root: string): string => {
   let rnVersion = '';
   try {
-    const pckJsonLocation = resolveCwd('react-native/package.json');
-    rnVersion = JSON.parse(fs.readFileSync(pckJsonLocation, 'utf-8')).version;
+    const pkgJsonLocation = resolveCwd(
+      path.join(root, 'node_modules', 'react-native', 'package.json'),
+    );
```

Comment:
@Krizzu let me know if that's ok

Reply:
Looks good to me

Reply:
https://github.com/sindresorhus/resolve-from